code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -991,6 +991,19 @@
 		if(alien != IS_TAJARA && prob(5))
 			H.delayed_vomit()
 
+/datum/reagent/drink/milk/fatshouter
+	name = "Fatshouter's Milk"
+	id = "fatshouter_milk"
+	description = "An opaque white liquid produced by the mammary glands of mammals, probably the fatshouter from Adhomai."
+	color = "#DFDFDF"
+	taste_description = "fatty milk"
+
+	glass_icon_state = "glass_white"
```

Comment:
This and all vars after this are not needed, remove them.

Reply:
Done.